plugins/woocommerce/changelog/update-pricing-inputs-width

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Change the width of pricing fields to 50% in big screens (>960px) in new product management experience
```

Comment:
This should be updated since it is not just the pricing fields that are being updated.